assets/python/ml/templates/model.pyi

```diff
@@ -111,5 +111,11 @@ with settings.context as context:
         # Make some predictions
         predictions = model.predict(descriptors)
 
+        {% if category == "classification" %}
+        # Transform predictions back to their original labels
+        label_encoder: sklearn.preprocessing.LabelEncoder = context.load("label_encoder")
```

Comment:
Not sure I understand what this line actually does - pls explain? The ":" notation before "=" specifically.

Reply:
It's type-hinting notation, to help the IDE with auto-completion. The general syntax goes:

`some_variable : type_of_the_variable = some_expression`

It's useful for cases like this, where `context.load` can return many different types, but we know it returns a LabelEncoder here.